### DISCUSSION TITLE
issue with v2 migration (missing token accounts)

### DISCUSSION THREAD

**Discussion by stefanosc** [wormhole-foundation/wormhole#464] [Problems / bugs] [answered]
Hello, I just tried using v2 to move some wrapped ETH sollet from solana to ethereum.
I use phantom 
I get a message I need to migrate 
I land on the migrate page and I only see an error message that says
`You have not created the necessary token accounts.`

There is no way to do anything.

**Comment by leoluk** [comment, marked as answer]
Is there a button allowing you to create the missing token accounts?

**Reply by stefanosc** [reply]
Hi no there is no button, please see screenshots I took

See the ETH present in the wallet:

<img width="638" alt="Screen Shot 2021-09-22 at 1 18 43 AM" src="https://user-images.githubusercontent.com/4052799/134260451-cb2854b1-143e-448a-8001-ce1e1a562a94.png">

It suggests migration:
<img width="569" alt="Screen Shot 2021-09-22 at 1 18 59 AM" src="https://user-images.githubusercontent.com/4052799/134260532-00ff41d8-6f40-4bf8-93bb-45dd4ba8b5f4.png">


I have to manually input the amount but nothing else happens (BTW major usability issue: there is no way to click to autofill the total, and it does not show me the balance, so I need to remember from the previous page and fill manually)
<img width="598" alt="Screen Shot 2021-09-22 at 1 19 20 AM" src="https://user-images.githubusercontent.com/4052799/134260740-88ba1f65-e790-4d75-b7cd-217dcb75e39b.png">


Let me know if you need anything else

**Reply by leoluk** [reply]
Thanks for the excellent feedback, we'll let you know if we need anything else!

**Reply by stefanosc** [reply]
I found a solution, I will DM you to share details

**Reply by leoluk** [reply]
Please post here - DMs are disabled

**Reply by leoluk** [reply]
(the team is also working to improve the UX here)

**Reply by stefanosc** [reply]
Please check your Twitter DM

**Reply by evan-gray** [reply]
Hi @stefanosc. I believe a fix was deployed which may resolve your issue. Could you please try again and let us know the result?

**Reply by stefanosc** [reply]
Hi @evan-gray yes it was resolved last night, but I believe it was due to a token migration I did through saber.
Please contact me in private for details of the TX if useful
